src/App.js

```diff
@@ -1,8 +1,7 @@
 import React from "react";
-import { Routes } from "./routes";
+import AppRoutes from "./components/Routes";
```

Comment:
Revet this back to the previous state:

```suggestion
import { Routes } from "./routes";
```